client/src/api/axios-config.js

```diff
@@ -41,12 +39,14 @@ const authRefresh = async (response) => {
   const status = response ? response.status : null
   if (status === 401) {
     return refreshIDToken().then((newToken) => {
-      if (newToken) {
+      if (newToken != null) {
```

Comment:
Why switch to newToken != null (or newToken !== null)?

Reply:
I think this is more specifically checking for null, whereas `if(newToken)` could be checking both for null and empty string, etc. Happy to change it back though if you think the previous check works better. But will definitely change it to `if(newToken !== null)` if we want to be explicit

Reply:
newToken should never be "" or 0, but it could be undefined. `if(newToken)` should suffice.

Reply:
Sounds good, will change it back then. Just out of curiosity, what conditions would `refreshIDToken()` return `undefined` for `newToken`?

Reply:
In the event that the Google endpoint for refreshing the idToken doesn't actually return an idToken field, `refreshIDToken() `could return undefined.

However, your destructuring code in `refreshIDToken()` (`.then(({ data: { id_token: idToken } }) => {`) might also cause an error to be thrown. Then, refreshIDToken() won't return `undefined`.